css/main.css

```diff
@@ -0,0 +1,210 @@
+*{
+    padding: 0;
+    margin: 0;
+}
+
+
+.container{
+    display: grid;
+    grid-template-columns: repeat(1, 1rf);
+    grid-template-rows: repeat(3, auto);
+    grid-gap: 1px;
+    grid-template-areas:"head"
+                        "main"
+                        "footer";
+}
+
+.header{
+    grid-area: head;
+    display: grid;
+    grid-template-columns: repeat(7,14.4%);
+    grid-gap: 1px;
+    
+}
+.nav{
+    background: #C5C5C5;
+    display: flex;
+    justify-content: center;
+    padding: 40px;
+    
+
+}
+.button{
+    border: 0px;
+    padding: 10px 40px;
+    background: gray;
+    color: white;
+}
+
+.main{
+    display: grid;
+    position: relative;
+    
+}
+
+.latest-articles{
+    display: grid;
+    grid-template-columns: repeat(3, 33.33%);
+    grid-template-rows: repeat(2,100%);
+    grid-gap: 2px;
+
+}
+.articles{
+    
+    height: 400px;
+
+    display: flex;
+    justify-content: center;
+    align-items: flex-start;
+    background: gray;
+    padding-top: 150px
+}
+
+
+.float-title{
+    position: absolute;
+    top: 10px;
+    left: 30%;
+    padding: 10px;
+    text-align: center;
+}
+
+.float-title p{
+    background: white;
+    padding: 10px;
+    font-weight: 600;
+
+
+}
+.middle-p{
+    width: 35%;
+    margin: auto;
+    border-top: 0px;
+    
+}
+
+.middle{
+    position: relative;
+    
+}
+
+
+.latest-posts-three{
+    display: grid;
+    grid-template-columns: repeat(4,25%);
+    height: 300px;
+    grid-gap: 2px;
+}
+
+.add{
+    background: #AEAEAE;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.posts{
+    background: #DEDEDE;
+        display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.banner{
+    margin: 2px;
+    display: grid;
+    grid-template-columns: 50% 25% 25%;
+    height: 300px;
+    grid-gap: 2px;
+}
+
+
+.normal-banner{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.latest-posts{
+    display: grid;
+    grid-template-columns: 25% 25% 50%;
+    height: 300px;
+    grid-gap: 2px;
+}
+
+.large-banner{
+    width: 100%;
+    height:900px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background: #6A6A6A;
+    color: white;
+}
+
+
+.person-week{
+    display: grid;
+    grid-template-columns: repeat(2, 50%);
+    grid-auto-rows: repeat(4,auto);
```

Comment:
same here auto shouldn't be repeat

Reply:
same here can be omitted, doesn't have any function.

Reply:
The change has been committed and everything is fixed